bundle/src/main/java/com/adobe/acs/tools/dumplibs/DumplibsParams.java

```diff
@@ -0,0 +1,82 @@
+package com.adobe.acs.tools.dumplibs;
+
+import com.adobe.granite.ui.clientlibs.LibraryType;
+import org.apache.commons.lang.StringUtils;
+import org.apache.sling.api.SlingHttpServletRequest;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class DumplibsParams {
+
+    private static final Logger log = LoggerFactory.getLogger(DumplibsParams.class);
+
+
+    private static final String PARAM_CATEGORIES = "categories";
+    private static final String PARAM_PATH = "path";
+    private static final String PARAM_TYPE = "type";
+    private static final String PARAM_TRANS = "trans";
+    private static final String PARAM_THEMED = "themed";
+
+    private String[] categories;
+    private String path;
+    private LibraryType type;
+    private boolean trans;
+    private boolean themed;
+
+    DumplibsParams(SlingHttpServletRequest request){
+        path = request.getParameter(PARAM_PATH);
+        categories = getCatigoriesArray(request.getParameter(PARAM_CATEGORIES));
+
+        themed = parseBoolean(request.getParameter(PARAM_THEMED));
+        trans = parseBoolean(request.getParameter(PARAM_TRANS));
+        type = getLibraryType(request.getParameter(PARAM_TYPE));
+    }
+
+    private  String[] getCatigoriesArray(String catigs){
```

Comment:
There is a typo in the method name. It should be getCategoriesArray() instead of getCatigoriesArray()

Reply:
@muhammad-naveed good catch! I keep typing 1categories` as `catigories` for some reason, and I keep going back to fix it.. smh

Anyway, fixed it!